modules/renter/downloadcache_test.go

```diff
@@ -0,0 +1,39 @@
+package renter
+
+import (
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+)
+
+// TestAddChunkToCache tests that the oldest chunk is removed
+func TestAddChunkToCache(t *testing.T) {
+	udc := &unfinishedDownloadChunk{
+		download: &download{
+			staticDestinationType: "httpseekstream",
+		},
+		chunkCache: make(map[string]*cacheData),
+		cacheMu:    new(sync.Mutex),
+	}
+
+	data := []byte{1, 2, 3, 4}
+
+	// Fill Cache
+	for i := 0; i < downloadCacheSize; i++ {
+		udc.staticCacheID = strconv.Itoa(i)
+		udc.addChunkToCache(data)
```

Comment:
You don't really need to declare `data` since it also works with an empty data slice.
`udc.addChunkToCache([]byte{})` should work just fine.